docs/contributing/dependencies/index.md

```diff
@@ -67,31 +69,45 @@ typically group dependencies we know are related and should be upgraded at the s
 keep groups as small as possible to minimize the impact and increase confidence in approving and
 merging.
 
+### Jira ticket
+
+A Jira ticket will automatically be created for each Renovate PR that is opened. It will be assigned
+to the appropriate team based on the dependency ownership.
+
+The Jira ticket should be used to track the work through sprint planning, prioritization, review,
+and testing.
+
 ### Review
 
 A typical dependency workflow involves the following steps:
 
-1. Read the proposed changes.
-2. Review the release notes of each dependency, for each released version between the current and
+1. Move the Jira ticket to In Progress.
+2. Read the proposed changes.
+3. Review the release notes of each dependency, for each released version between the current and
    the proposed upgrade. Identify if there are any deprecations or breaking changes affecting our
    code.
-   1. For breaking changes, either resolve them yourself, or for major changes, coordinate with the
-      other teams.
-   2. For deprecations, create high priority Jira tickets on the affected teams' backlogs with a due
-      date at least one sprint before the next scheduled major release of the dependency.
-3. Verify CI status.
-4. If test coverage is lacking, check out locally and manually confirm a few key areas.
-5. Review the proposed code changes and approve the PR.
-6. Write a Jira ticket containing testing notes for QA.
-7. Merge the PR. Assign the Jira ticket to QA.
+   1. For **breaking changes**, either resolve them yourself, or for major changes, coordinate with
+      the other teams.
+   2. For **deprecations**, create high priority Jira tickets on the affected teams' backlogs with a
+      due date at least one sprint before the next scheduled major release of the dependency.
+4. Verify CI status.
+5. If test coverage is lacking, check out locally and manually confirm a few key areas.
+6. Review the proposed code changes and approve the PR.
+7. Write a Jira ticket containing testing notes for QA.
+8. Merge the PR.
+9. Assign the Jira ticket to QA.
```

Comment:
❓ We're QAing these now?

Reply:
Yes, to follow our SDLC.

Reply:
Also discussed, but the win is to not need to "QA" these, especially bumps that should guarantee existing functionality and nothing break.